.github/workflows/vendor_gardener.yaml

```diff
@@ -0,0 +1,31 @@
+name: Vendor Gardener
+on:
+  push:
+    branches:
+    - dependabot/go_modules/github.com/gardener/**
+  pull_request:
+    branches:
+    - dependabot/go_modules/github.com/gardener/**
+permissions: write-all
+jobs:
+  run:
+    name: Run make revendor
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '^1.19.7'
```

Comment:
Can we somehow use a go version without specifying the patch version? Something like go1.19 (get the latest patch release available).

Reply:
What do you think about [this](https://github.com/marketplace/actions/go-version-action?version=v1#examples) solution?

Reply:
Isn't this yet external plugin (`arnested/go-version-action@v1`)?

You should be able to do:
```suggestion
        go-version: '1.19'
        check-latest: true
```

See https://github.com/actions/setup-go#using-stableoldstable-aliases

Reply:
[This](https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file) seems even better because no go version should be supplied

Reply:
Okay, let's go on with it.